src/composables/themedLogo.js

```diff
@@ -0,0 +1,12 @@
+import { computed } from 'vue';
+
+export function useThemedLogo() {
+  const originalSrc = '/img/logo';
+  const themedSuffix = '-spooky';
```

Comment:
Eventually this can be a more sophisticated time-based calculation that is itself computed, but for right now I think it's reasonable to just hard code the suffix. The main reason for the composable as-is is to centralize references without replacing the file itself so that we can update this one line to change the images that are consuming the themed logo

Reply:
All good, we should brainstorm some restrictions around timing, length, etc.